### DISCUSSION TITLE
Build a Auth screen with rest

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#17394] [Questions] [answered]
Hi members, how to make ability to update token in request headers using $fetch when new user was log

**Comment by Flanker72** [comment, marked as answer]
Hi!
You can do this using different aproaches like:
1. Create your own wrapper on the fetch/useFetch, and put headers there dynamically from some kind of state (useState/pinia/cookies)
2. Use dynamic reactive helpers like [useCookie](https://v3.nuxtjs.org/api/composables/use-cookie) from Nuxt or [useLocalStorage](https://vueuse.org/core/useLocalStorage/) from vueuse
